src/modules/75-cf/components/FlagActivation/ServicesList.tsx

```diff
@@ -97,13 +109,13 @@ const EditServicesModal: FC<EditServicesProps> = ({
 
       {serviceError && <Page.Error message={getErrorMessage(serviceError)} onClick={refetchServices} />}
 
-      {allServices && !serviceError && (
+      {filteredServices && !serviceError && (
```

Comment:
since you've defaulted `filterServices` to `[]` , this will always return true. Think you want to check the length? 

```suggestion
      {filteredServices.length && !serviceError && (
```

You could even just get rid of this altogether since `.map()` wont do anything on an empty array

```suggestion
      {!serviceError && (
```